xclim/indices/_threshold.py

```diff
@@ -398,19 +394,14 @@ def snd_season_end(
 @declare_units(snw="[mass]/[area]", thresh="[mass]/[area]")
 def snw_season_end(
     snw: xarray.DataArray,
-    thresh: Quantified = "20.00 kg m-2",
+    thresh: Quantified = "4 kg m-2",
     window: int = 14,
     freq: str = "YS-JUL",
 ) -> xarray.DataArray:
-    r"""End date of continuous snow water cover.
+    r"""End date of continuous snow cover.
```

Comment:
Nitpicking: Following later changes, should we write: "snow amount cover" here? Or amount + cover looks weird?